jme3-core/src/main/java/com/jme3/scene/shape/Cylinder.java

```diff
@@ -127,10 +124,10 @@ public Cylinder(int axisSamples, int radialSamples,
      * mapped to texture coordinates (0.5, 1), bottom to (0.5, 0). Thus you need
      * a suited distorted texture.
      *
-     * @param axisSamples
-     *            Number of triangle samples along the axis.
-     * @param radialSamples
-     *            Number of triangle samples along the radial.
+     * @param axisSamples The number of vertices samples along the axis. It is equal to the number of segments + 1; so 
+	 * that, for instance, 4 samples mean the cylinder will be made of 3 segments.

```

Comment:
Sorry about all that.
I usually use tabs and I did not check that it was the case, my bad.